content/topics/Visualization/data-visualization/graphs-charts/time-series-ggplot2.md

```diff
@@ -0,0 +1,381 @@
+
+Here's a basic syntax outline:  
+ggplot(data = your_data_frame, aes(x = date_column, y = values_column)) +
+  geom_line()
+  
+In this structure:
+- your_data_frame is your dataset.
+- date_column and values_column denote the columns of your dates and your variable of interest, respectively.
+  
+Consider this practical example for visualizing Apple (AAPL) stock prices post-January 1, 2014:
+
+{{% codeblock %}}
+
+```R
+# Filter the data for Apple (AAPL) stock prices after January 1, 2014
+AAPL <- prices %>%
+  filter(symbol == "AAPL" & date > as.Date("2014-01-01"))
+
+# Create a line chart using ggplot2
+ggplot(data = AAPL, aes(x = date, y = adjusted)) +
+  geom_line() + 
+  #  customization (Shown in the source code)
+```
+
+{{% /codeblock %}}
+
+<p align = "center">
+<img src = "../images/time-series-1.png" width="400">
+</p>
+
+For a more in depth view, the `geom_point` function can introduce data points to your line chart. However, with numerous points, readability may decrease.
+
+{{% codeblock %}}
+
+```R
+AAPL_point <- prices %>%
+  filter(symbol == "AAPL" & date > as.Date("2014-01-01") & date < as.Date("2014-06-01"))
+
+# Line chart with data points
+ggplot(data = AAPL, aes(x = date, y = adjusted)) +
+  geom_line() +
+  geom_point() +
+  # Customization
+```
+
+{{% /codeblock %}}
+
+<p align = "center">
+<img src = "../images/time-series-2.png" width="400">
+</p>
+
+{{% tip %}}
+
+**Tip for Date Formatting**:   
+To ensure your dates are correctly plotted, they should be in the `Date` class. The `lubridate` package is the best practice for date conversions, alongside the `as.Date()` or `as.POSIXct()` functions. Confirm your date column's format with `class()` or `str()` checks.
+
+{{% /tip %}}
+
+To incorporate a shaded area beneath the time series use the `geom_area()` function, this adds a visual layer of depth and context to the plot:
+
+{{% codeblock %}}
+
+```R
+# Create a line chart with shaded area using ggplot2
+ggplot(data = AAPL, aes(x = date, y = adjusted)) +
+  geom_area(fill = "lightgrey", alpha = 0.5) +  # Add shaded area with transparency
+  geom_line() +  # Add line plot
+  # Customization
+  ```
+
+{{% /codeblock %}}
+
+<p align = "center">
+<img src = "../images/time-series-3.png" width="400">
+</p>
+
+## Adjusting Dates on the X-Axis with scale_x_date
+Building on `ggplot2`'s capability to automatically recognize date formats, adjusting dates on the X-axis becomes valuable when the default settings do not convey the most valuable or easy-to-interpret information. The `scale_x_date()` function in `ggplot2` provides control over how date scales appear on the X-axis of time series plots. It enables customization of the date format displayed, in addition to defining intervals and setting specific limits within your data.
+
+**Syntax Overview:**  
+*scale_x_date(date_breaks = "interval", date_labels = "strftime_format", limits = NULL)*
```

Comment:
Since you point out a correct syntax here, what do you think of using code snippet here?  Italic does not look natural for a piece of code to me.